doc/adoc/user_guide.adoc

```diff
@@ -163,6 +163,9 @@ preserve:
   rules:
     - /etc/udev/rules.d/a.rules
     - /etc/udev/rules.d/b.rules
+  static:
+    - /etc/sysconfig/proxy
+    - /path/to/be/preserved/file
```

Comment:
There should be some words about the difference between ```rules:``` and ```static:``` I think it would be good to mention that files preserved as udev rules will also make the DMS to reload udev and its rules to make the new rule set effective, whereas the files in the static section are just copied without any further action

Reply:
Done. Once all is good, I will squash the commits into one.

Reply:
This is great now :+1: 

Just go ahead and merge after squash. NOTE: Please remember we need a bugzilla number for this to avoid trouble with maintenance. If there is no bugzilla report we can refer this to, please add a bug in bugzilla and use the number here in the commit message as "bsc#NR" Thanks much